### PR TITLE
Update mempool app to v2.1.2

### PR DESCRIPTION
**Pull request by bguillaumat**
```
This is a patch release that optimizes memory usage for embedded devices like Raspberry Pi

Notes:

The disk cache now has its own folder, so mv cache*.json ./cache/ before restart to migrate
Changes:

Optimize memory usage when writing disk cache (#342)
Reduce backend maximum heap size setting to 2G (#345)
Enable mempool clear protection on all backends (#335)
Make mempool clear protection timeout configurable (#343)
Minor tweaks to About page text, links (#350)
Logo design update (#349) by @pedromvpg
Fix style on block hover (#347) by @Czino and @Eric-Machinas
```

### CODE REVIEW

**Comment by downet** [comment]
Is it possible to make also the logging non debug? As I saw it's writing all debug info towards sd card.

File: apps/mempool/docker-compose.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ services:
       MYSQL_DATABASE: "mempool"
       MYSQL_USER: "mempool"
       MYSQL_PASS: "mempool"
-      CACHE_DIR: "/backend/cache/"
```
Comment (lukechilds) [inline apps/mempool/docker-compose.yml:47]: Why is this change needed?

**Comment by lukechilds** [comment]
Thanks for this @bguillaumat!

>The disk cache now has its own folder, so mv cache*.json ./cache/ before restart to migrate

Do we need to do any special handling on our end for this or will it not affect us if we're already passing in an explicit `CACHE_DIR ` path?

**Comment by bguillaumat** [comment]
@lukechilds nah it's all good for us it's an update Simon made for old mempool to be compliant with the cache_dir var

File: apps/mempool/docker-compose.yml (hunk 1 above)
Comment (bguillaumat) [reply apps/mempool/docker-compose.yml:47]: They've update mempool so now their code add this var to "/cache.json" so I've removed the useless slash on the docker-compose

**Comment by lukechilds** [comment]
>Is it possible to make also the logging non debug? As I saw it's writing all debug info towards sd card.

@downet I think a better solution is for us to redirect Umbrel logs to the SSD. I'll work on that soon.

File: apps/mempool/docker-compose.yml (hunk 1 above)
Comment (lukechilds) [reply apps/mempool/docker-compose.yml:47]: Ahh, good spot!

**Review by lukechilds — APPROVED** [review]

**Comment by lukechilds** [comment]
Tested and works perfectly, thanks @bguillaumat!
